java/main/com/google/privacy/differentialprivacy/GaussianNoise.java

```diff
@@ -138,14 +144,39 @@ public ConfidenceInterval computeConfidenceInterval(
    */
   @Override
   public ConfidenceInterval computeConfidenceInterval(
-      long noisedX,
-      int l0Sensitivity,
-      long lInfSensitivity,
-      double epsilon,
-      Double delta,
-      double confidenceLevel) {
-    // TODO: Implement confidence interval computation.
-    throw new UnsupportedOperationException("Not implemented yet.");
+          long noisedX,
+          int l0Sensitivity,
+          long lInfSensitivity,
+          double epsilon,
+          Double delta,
+          double alpha) {
+    CheckConfidenceIntervalParameters(l0Sensitivity, lInfSensitivity, epsilon, delta, alpha);
+    double l2Sensitivity = lInfSensitivity * Math.sqrt((double) l0Sensitivity);
+    double sigma = getSigma(l2Sensitivity, epsilon, delta);
+    ConfidenceInterval confInt = computeConfidenceInterval((double) noisedX, sigma, alpha);
+    confInt =
+            ConfidenceInterval.create(
+                    Math.round(confInt.lowerBound()), Math.round(confInt.upperBound()));
+    return confInt;
+  }
+
+  /**
+   * Returns {@link ConfidenceInterval} gaussian object with 1-{@code alpha} confidence level and
```

Comment:
nit: Is `gaussian` here necessary?

Reply:
Hm, since it's in the GaussianNoise document i think you're right, it's not.